src/main/requestInitialServerData.js

```diff
@@ -0,0 +1,16 @@
+import { homeNarrow, specialNarrow } from '../utils/narrow';
+import { focusPing } from '../api';
+
+export default ({
+  auth,
+  fetchEvents,
+  fetchUsersAndStatus,
+  backgroundFetchMessages,
+  fetchMessages,
+}) => {
+  fetchEvents(auth);
+  fetchUsersAndStatus(auth);
+  fetchMessages(auth, Number.MAX_SAFE_INTEGER, 20, 0, homeNarrow(), [true, true]);
```

Comment:
Looks like there was a merge conflict here. `[true, true]` should be `{ older: true, newer: true }`